pkg/descheduler/strategies/topologyspreadconstraint.go

```diff
@@ -272,8 +271,7 @@ func balanceDomains(
 		// also (just for tracking), add them to the list of pods in the lower topology
 		aboveToEvict := sortedDomains[j].pods[len(sortedDomains[j].pods)-movePods:]
 		for k := range aboveToEvict {
-			if err := validatePodFitsOnOtherNodes(aboveToEvict[k], nodeMap); err != nil {
-				klog.V(2).InfoS(fmt.Sprintf("ignoring pod for eviction due to: %s", err.Error()), "pod", klog.KObj(aboveToEvict[k]))
+			if !node.PodFitsAnyOtherNode(ctx, client, aboveToEvict[k], nodes) {
 				continue
```

Comment:
```suggestion
				klog.V(2).InfoS("ignoring pod for eviction as it does not fit on any other node", "pod", klog.KObj(aboveToEvict[k]))
				continue
```

Reply:
@a7i Good point. I removed that because `PodFitsAnyOtherNode` logs these messages. I realize now that we should also log here at a lower level than in `PodFitsAnyOtherNode` in order to log what the strategy is doing.